src/components/SharedComponents/SettingsComponent.vue

```diff
@@ -95,16 +105,16 @@
         dataToggle: false,
         modalTitle: 'Welcome to a Program Wars!',
         localPlayers: [{name: '', isAi: false}],
+        storedPlayers: [],
         newPlayer: '',
         gameStart: false,
-        selected: '35',
+        selected: '50',
```

Comment:
Based on what I was seeing with kids this morning, 50 might make the game too long. Let's leave it at 35.